### DISCUSSION TITLE
Spyne 2.14 production ready?

### DISCUSSION THREAD

**Discussion by mastantm** [arskom/spyne#698] [Q&A] [answered]
The spyne 14.0 readme states
```
WARNING: This is from spyne's development branch. This version is not released yet! Latest stable release can be found in the 2_13 branch.
```
Is this an oversight or factual? Thanks!

**Comment by plq** [comment, marked as answer]
Hey, thanks for asking. Yes 2.14 is production-ready and is in fact running in production in all our projects. I removed the misleading sentence from the readme.
